src/Files.App/UserControls/StatusBarControl.xaml

```diff
@@ -130,6 +138,14 @@
 				Text="{x:Bind SelectedItemsPropertiesViewModel.SelectedItemsCountString, Mode=OneWay}"
 				Visibility="{x:Bind SelectedItemsPropertiesViewModel.IsItemSelected, Mode=OneWay}" />
 
+			<!--  Divider  -->
+			<Border
+				x:Name="ItemSizeDivider"
+				Width="1"
+				Height="18"
+				x:Load="{x:Bind SelectedItemsPropertiesViewModel.ItemSizeVisibility, Mode=OneWay}"
+				Background="{ThemeResource ControlStrokeColorDefaultBrush}" />
```

Comment:
```suggestion
				Background="{ThemeResource DividerStrokeColorDefaultBrush}" />
```

DividerStrokeColorDefaultBrush is prefered DividerStrokeColorDefaultBrush is unvisible when system theme is dark.

Reply:
👍